src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/PaymentsProcessor.java

```diff
@@ -60,11 +60,12 @@ public void updatePayments(CaseDetails exceptionRecord, long newCaseId) {
             );
 
         if (containsPayments) {
+
+            log.info("Contains Payments, send payment update message. ER id: {}", exceptionRecord.getId());
```

Comment:
Also can we please add jurisdiction payment Dcn to this log ?

Reply:
do we always have payment Dcns ?